inscriptions/ISic000388.xml

```diff
@@ -185,18 +190,21 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="EDR" n="EDR141197" xml:lang="la">
+            <div type="edition" xml:space="preserve" subtype="EDR" n="EDR141197" xml:lang="la" resp="#JH">
                 <ab>
-                    <lb n="1"/>
-                    S A O<lb n="2"/>C I
+                    <lb n="1"/>S · A · O
```

Comment:
complex case this one. So, I would encode this simply with `<abbr>` around each letter (i.e. unresolvable abbreviation)